internal/mocks/service.go

```diff
@@ -17,6 +18,9 @@ func NewMockSBOMService(response response, assertions ...func(r *http.Request))
 		}
 
 		w.Header().Set("Content-Type", response.contentType)
+		if response.status >= 100 {
```

Comment:
MINOR: is it possible to use the `http` constant?

Reply:
done!